ui/src/app/cohort-search/search-group-list/search-group-list.component.ts

```diff
@@ -23,7 +23,7 @@ export class SearchGroupListComponent {
   }
 
   get title() {
-    const verb = this.role.charAt(0).toUpperCase() + this.role.slice(1, -1) + 'd';
-    return `${verb} Participants`;
+    const verb = this.role.charAt(0).toUpperCase() + this.role.slice(1, -1);
```

Comment:
In retrospect, we can probably drop this line and just use the `titlecase` pipe in the template.  E.g.

```
<h3>{{ title | titlecase }}</h3>
```

Should do the trick.

Reply:
Done.